src/main/proto/README.md

```diff
@@ -5,6 +5,8 @@ Marathon is using Google [Protocol Buffers](https://developers.google.com/protoc
 ## Dependencies
 
 - Google [Protocol Buffers](https://developers.google.com/protocol-buffers) v2.5 should be installed on the system.
+-- Note: protoc 2.6 (and higher) will generate incompatible code. Make sure your version of protoc is 2.5.
```

Comment:
This change is unrelated to the PR, please create a separate PR for this as it would otherwise be lost if we ever have to revert this PR.